electron-builder.json

```diff
@@ -33,7 +33,7 @@
   "mac": {
     "category": "public.app-category.video",
     "target": "dmg",
-    "icon": "dist/favicon",
+    "icon": "src/assets/icons.icns",
```

Comment:
Just a note, if you apply the same icon fix to linux, it builds for me! 👍 

I will test out the built image when I get home! 😄

Reply:
Don't forget to change this! 👍